skimage/transform/tests/test_warps.py

```diff
@@ -331,6 +331,22 @@ def test_resize3d_bilinear():
     assert_almost_equal(resized, ref)
 
 
+def test_resize_dtype():
+    x = np.zeros((5, 5))
+    x_f32 = x.astype(np.float32)
+    x_u8 = x.astype(np.uint8)
+    x_b = x.astype(bool)
```

Comment:
I'm not sure we care about supporting boolean data types. @jni?

Reply:
Eh. If it works, why not keep it working?